sirepo/template/hellweg.py

```diff
@@ -98,17 +99,16 @@ def extract_beam_report(report, run_dir, frame):
 
 
 def extract_parameter_report(report, run_dir):
-    from rslinac.solver import BeamSolver
-    solver = BeamSolver(
+    beam_solver = solver.BeamSolver(
```

Comment:
I don't find that naming the instance `beam_solver` does much, especially with all the repetition. You could just call it `s`, and get as much clarity in the code.